core/templates/dev/head/pages/collection_player/CollectionPlayer.ts

```diff
@@ -16,6 +16,23 @@
  * @fileoverview Controller for the learner's view of a collection.
  */
 
+// vvv this block of requires should be removed vvv
```

Comment:
Should this be fixed? Ditto for other cases, we shouldn't leave TODOs in.

Reply:
This is needed until the extensions get migrated to webpack, I will include `TODO(vojtechjelinek)`.